CHANGELOG.md

```diff
@@ -14,3 +14,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Integrate support in Helm resources for users with and without WSO2 subscriptions
 
 [v6.5.0.2]: https://github.com/wso2/kubernetes-ei/compare/v6.5.0.1...v6.5.0.2
+
+## [V6.5.0.3] - 2019-08-15
```

Comment:
@Savidude let's keep this as `Unreleased` as in the example. We can set the date to the specific release date when we merge this branch with the `master` branch.

Reply:
Changed https://github.com/wso2/kubernetes-ei/pull/188/commits/b9d7d6197aebd3940b8a092b93f27b02c59cefb4